server/src/main/java/org/candlepin/resource/ConsumerResource.java

```diff
@@ -596,11 +596,18 @@ else if (update.getFact("distributor_version") !=  null) {
      */
     private void checkConsumerName(Consumer consumer) {
         // for now this applies to both types consumer
-        if (consumer.getName() != null &&
-            consumer.getName().indexOf('#') == 0) {
-            // this is a bouncycastle restriction
-            throw new BadRequestException(
-                i18n.tr("System name cannot begin with # character"));
+        if (consumer.getName() != null) {
+            if (consumer.getName().indexOf('#') == 0) {
+                // this is a bouncycastle restriction
+                throw new BadRequestException(
+                    i18n.tr("System name cannot begin with # character"));
+            }
+
+            int max = Consumer.MAX_LENGTH_OF_CONSUMER_NAME;
```

Comment:
@Ceiu : I know. Checkstyle was shouting on me, because of length of line, when local variable was inlined.